sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -59,29 +94,31 @@ def _populate_metric_results(self, response):
     # Get the tentative/committed versions of every metric together.
     metrics_by_name = defaultdict(lambda: {})
     for metric in user_metrics:
+      if (metric.name.name.endswith('(DIST)') or
+          metric.name.name.endswith('[MIN]') or
+          metric.name.name.endswith('[MAX]') or
+          metric.name.name.endswith('[MEAN]') or
+          metric.name.name.endswith('[COUNT]')):
+        warn('Distribution metrics will be ignored in the MetricsResult.query'
+             'method. You can see them in the Dataflow User Interface.')
+        # Distributions are not yet fully supported in this runner
+        continue
       tentative = [prop
                    for prop in metric.name.context.additionalProperties
                    if prop.key == 'tentative' and prop.value == 'true']
-      key = 'tentative' if tentative else 'committed'
-      metrics_by_name[metric.name.name][key] = metric
+      is_tentative = 'tentative' if tentative else 'committed'
```

Comment:
Is `is_tentative` a string of "tentative", or "committed" and `is_tentative` a boolean? If that is the case maybe their names could be swapped.